CefSharp.WinForms.Example/Handlers/MenuHandler.cs

```diff
@@ -10,16 +10,24 @@ void IContextMenuHandler.OnBeforeContextMenu(IWebBrowser browserControl, IBrowse
         {
             //To disable the menu then call clear
             // model.Clear();
+
+            bool removed = model.Remove(132); // Remove "View Source" option
```

Comment:
(That link is slightly outdated, will need to look in the `include` files for up to date values).